test/models/user_test.rb

```diff
@@ -1,6 +1,6 @@
 require 'test_helper'
 
-class UserTest < ActiveSupport::TestCase
+class UserTest < ActiveSupport::TestCase # rubocop:disable Metrics/ClassLength
```

Comment:
@hpjaj, should we break this up into smaller segments?

Reply:
No, let's keep all of the user model related test code in this file.  

As far as disabling RuboCop, you can:

- do it like this: http://www.mindonrails.com/notes/334-disabling-and-re-enabling-a-rubocop-rule-for-a-specific-set-of-code
- add an exclusion for the test file to `.rubocop.yml`

The latter is slightly preferable.